content/en/docs/instrumentation/go/getting-started.md

```diff
@@ -79,14 +79,44 @@ func rolldice(w http.ResponseWriter, r *http.Request) {
 
 Build and run the application with the following command:
 
-```sh
+```shell
 go run .
 ```
 
 Open <http://localhost:8080/rolldice> in your web browser to ensure it is
 working.
 
-## Instrumentation
+## Add OpenTelemetry Instrumentation
+
+Now we'll show how to add OpenTelemetry instrumentation to the sample app. If
+you are using your own application, you can follow along, just note that your
+code may be slightly different.
+
+### Add Dependencies
+
+Install the following packages:
+
+```shell
+go get "go.opentelemetry.io/otel" \
+  "go.opentelemetry.io/otel/exporters/stdout/stdoutmetric" \
+  "go.opentelemetry.io/otel/exporters/stdout/stdouttrace" \
+  "go.opentelemetry.io/otel/sdk/metric" \
+  "go.opentelemetry.io/otel/sdk/resource" \
+  "go.opentelemetry.io/otel/sdk/trace" \
+  "go.opentelemetry.io/otel/semconv/v1.21.0" \
+  "go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp"
+```
+
+This installs OpenTelemetry SDK components and `net/http` instrumentation.
+
+If you're instrumenting a different library for network requests, you'll need to
+install the appropriate instrumentation library. See
+[libraries](/docs/instrumentation/go/libraries/) for more info.
```

Comment:
I am not sure if we need to write it.

`go mod tidy` installs missing Go modules.

I think it is good to minimize the steps if they are not needed.

```suggestion
```

Reply:
Mmm, I think it's better to install up-front. Otherwise highly-used editors like go/vscode won't work well
<img width="802" alt="image" src="https://github.com/open-telemetry/opentelemetry.io/assets/6309070/4e04e1c3-004c-496e-b15f-1e79ca68cded">